src/megatron/bridge/recipes/nemotronh/nemotron_3_nano.py

```
@@ -168,8 +208,55 @@ def nemotron_3_nano_sft_config() -> ConfigContainer:
+        apply_query_key_layer_scaling=False,
+        persist_layer_norm=True,
+        attention_softmax_in_fp32=False,
+        first_last_layers_bf16=True,
+        is_hybrid_model=True,
+        moe_aux_loss_coeff=0.0001,
+        moe_router_score_function="sigmoid",
+        moe_router_enable_expert_bias=True,
+        moe_router_load_balancing_type="seq_aux_loss",
+        moe_router_dtype="fp32",
+        moe_grouped_gemm=True,
+        moe_token_dispatcher_type="alltoall",
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to argument: "moe_token_dispatcher_type"

(S106)